.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 **/*.rs.bk
+.idea
```

Comment:
Speaking as a fellow IntelliJ user, I would suggest you configure something like this in your local `~/.gitconfig`:

```
[core]
  excludesfile = /home/myuser/.gitignore
```

Using local excludes means that upstream projects don't have to track editor-specific ignore configuration for every editor under the sun.